pageserver/src/http/routes.rs

```diff
@@ -1623,6 +1638,7 @@ pub fn make_router(
     Ok(router
         .data(state)
         .get("/v1/status", |r| api_handler(r, status_handler))
+        .get("/v1/ready", |r| api_handler(r, ready_handler))
```

Comment:
```suggestion
        .get("/v1/ready", |r| testing_api_handler(r, ready_handler))
```

I don't think we are expecting anyone to use this except from tests.

Reply:
Unsure if this path (`/v1/ready`) should be called something else; for requests we are ready right as the http starts accepting requests.

Reply:
> how about /v1/wait_tenants_loaded ?

Perhaps just `/v1/tenants_loaded`. Wait feels like it implies some sort of action being performed or POST method being used for me.

> if the handler is going to be behind testing feature should i gate init_done_rx field in the State behind it as well ?

good point, yes. It would be good to rename it in `http::State` as `tenants_loaded` instead of `init_done_rx` as well. Ok to continue using `init_done_rx` in the pageserver/src/bin/pageserver.rs however, but in case someone looks at the `http/routes.rs` only, then it will make more sense to see `State::tenants_loaded` which is being waited on.

Reply:
I have addressed your comments